packages/trader/build/webpack.config.js

```diff
@@ -47,12 +47,14 @@ module.exports = function (env) {
                 '@deriv/deriv-charts': '@deriv/deriv-charts',
                 '@deriv/account': '@deriv/account',
                 '@deriv/reports': '@deriv/reports',
+                '@deriv/analytics': '@deriv/analytics',
```

Comment:
```suggestion
```
@akmal-deriv Don't need to add it here, `@deriv/analytics` doesn't build itself, `@deriv/trader` will bundle it 🙇🏻

Reply:
@farzin-deriv I don't think so, this doesn't work without modifying webpack and the target component (contract-types) is completely blank if I remove these lines and import `Rudderstack` inside. I wouldn't touch this if it worked without it.

Reply:
This might be @deriv/analytics package related issue I'm guessing

Reply:
@adrienne-deriv Would you please help to check on this 🙏🏻